src/click/types.py

```diff
@@ -234,7 +234,7 @@ class Choice(ParamType):
 
     name = "choice"
 
-    def __init__(self, choices: t.Sequence[str], case_sensitive: bool = True) -> None:
+    def __init__(self, choices: t.Collection[str], case_sensitive: bool = True) -> None:
```

Comment:
Please revert this. Requiring a sequence was deliberate on my part, as the order may matter for display purposes.

Reply:
easy enough; force-pushed back to a0bbbf2